spec/cb/responses/job/search_v3_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe Cb::Responses::Job::SearchV3 do
+  
+  shared_examples 'search_v3_response' do |example_response_file|
+
+    let(:search_response_hash) { YAML.load open(example_response_file) }
+    let(:search_results) { Cb::Responses::Job::SearchV3.new(search_response_hash) }
```

Comment:
since you've already described this on line 3 RSpec offers up `described_class` that you could use:
```ruby
described_class.new(search_response_hash)
```